### ISSUE TITLE
Bound axe acting like a bound pickaxe. clearing not only wood but terrain.

### ISSUE THREAD

**Issue by rhaejr** [WayofTime/BloodMagic#752]
#### Issue Description:

Note: If this bug occurs in a modpack, please report this to the modpack author. Otherwise, delete this line and add your description here. If this is a feature request, this template does not apply to you. Just delete everything.

This is part of a pack. Haven't tried it outside of the pack.
#### What happens:

Bound axe acting like a bound pickaxe. clearing not only wood but terrain.
#### What you expected to happen:

For the bound axe to clear wood.
#### Steps to reproduce:
1. Made multiple Bound Axes
2. 
3. 
   ...

---
#### Affected Versions (Do _not_ use "latest"):
- BloodMagic: 2.0.0-20
- Minecraft: 1.8.9
- Forge: 11.15.1764

**Comment by TehNut** [comment]
Is this reproducible in the latest 1.9 version?

**Comment by rhaejr** [comment]
Note sure we are playing the Simply Magic 2 pack. I looked around to see if the issue had come up before. We are definitely up for updating to the latest version if it won't break compatibility. Will Check.

**Comment by afghan2007** [comment]
I checked it out, I'm running blood magic with a few select mods like soul shard the old way.  I tried all my bound tools.  The bound axe leveled its area completely to include taking out sand and rock along with wood.  The shovel did the same, taking out wood and rock.  The pickaxes also cleared trees and dirt along with rock.

The 10k LP Boom doesn't seem to discriminate.  Otherwise, the tools behave like their diamond counterparts.  (Noticed that the bound sword say "Culling the weak" and doesn't have a 10k boom.  A [Suggestion] might be to add a boom like effect that only effect entities in front of the sword much like other bound gear doing just enough damage to kill some of the weaker mobs.
culling the weak".)

PS. I thought bound weapons were supposed to take durability damage but not break at the cost transferring damage to the LP network and needed to be repaired in the anvil with a bound agent dust or a mending enchant.

Blood magic: BloodMagic-1.9.4-2.0.0-39
Minecraft: 1.9.4
Forge: forge-1.9.4-12.17.0.1910-1.9.4-installer-win
